wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if perms.edit_private_notes and object.private_notes %}
+<h2>Private notes</h2>
+<p>(The following is not visible to submitters or attendees.)</p>
+<div class="well">
+  {{ object.private_notes|urlize|linebreaks }}
```

Comment:
I'm tempted to say we shouldn't display any notes here so that the individual talk pages closely resemble what is seen by ordinary users of the site (but happy for it to be included if the consensus is to display it).